src/main/kotlin/com/atlassian/performance/tools/report/api/junit/JUnitReport.kt

```diff
@@ -2,12 +2,14 @@ package com.atlassian.performance.tools.report.api.junit
 
 import com.atlassian.performance.tools.io.api.ensureDirectory
 import com.atlassian.performance.tools.io.api.resolveSafely
+import com.atlassian.performance.tools.jiraactions.api.ActionType
 import java.nio.file.Path
 
 interface JUnitReport {
 
     val testName: String
     val successful: Boolean
+    val action: ActionType<*>? get() = null
```

Comment:
We were using it to resolve tests parallelism problems we had with Gradle. We start to see the same problems for our integration tests. `JUnitReport` related classes could be moved to a separate module and used in other use cases. This change couples it with `jira-actions` and makes it JPT only util. If we'd like to solve the same problem for a different project, we'll have to duplicate JUnit reporting logic.